openedx/core/djangoapps/xblock/runtime/learning_core_runtime.py

```diff
@@ -360,7 +365,7 @@ def _lookup_asset_url(self, block: XBlock, asset_path: str) -> str | None:
         then this method will be called with asset_path="test.png" and should
         return a URL like:
 
-          http://studio.local.edly.io:8001/library_assets/cd31871e-a342-4c3f-ba2f-a661bf630996/static/test.png
+          http://local.openedx.io:8001/library_assets/cd31871e-a342-4c3f-ba2f-a661bf630996/static/test.png
```

Comment:
Err sorry that should be `http://studio.local.openedx.io:8001/` or `http://local.openedx.io:8000/`

Reply:
Right. I should know better. Is late, brain no-workie good. <sigh>